diff --git a/node_modules/progress-webpack-plugin/index.js b/node_modules/progress-webpack-plugin/index.js
index 9ad1d1f..e0998ee 100644
--- a/node_modules/progress-webpack-plugin/index.js
+++ b/node_modules/progress-webpack-plugin/index.js
@@ -125,14 +125,7 @@ function progressPlugin(options = {}) {
     }
   }
   return new Progress({
-    handler,
-    entries: false,
-    activeModules: false,
-    modules: true,
-    modulesCount: 5000,
-    dependencies: false,
-    dependenciesCount: 10000,
-    percentBy: 'entries'
+    handler
   })
 }
 
